### PR TITLE
fix(feature/bug): fixed no translation and minor UI changes 896

### LINKED ISSUE #896: Footer Bug / Feature request doesnt translate
Footer Bug / Feature request doesn't translate to ESP

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- Resolve #896 #898 

### Steps to test

1. See footer and change language + minor UI text changes

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/components/Footer/index.js [hunk 1]
```diff
@@ -60,25 +60,17 @@ const Footer = () => {
...
-                      {link.text}
-                    </a>
-                  }
-                />
-              </ListItem>
-            ))}
+          <Link
+              href="https://github.com/eoscostarica/eosio-dashboard/issues/new/choose"
+              onClick={preventDefault}
+              style={{ textDecoration: 'none' }}
```
Comment (tetogomez) [inline webapp/src/components/Footer/index.js:66]: avoid inline style
please create a class and use it!

**Review by tetogomez — CHANGES_REQUESTED** [review]

File: webapp/src/components/Footer/index.js (hunk 1 above)
Comment (fvives99) [reply webapp/src/components/Footer/index.js]: fixed

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**